docs/tutorial_2_new_converter.rst

```diff
@@ -42,6 +42,7 @@ an example.
 
     auto_tutorial/plot_icustom_converter
     auto_tutorial/plot_jcustom_syntax
+    auto_tutorial/plot_function_transformer
```

Comment:
Nit:
Move it up so the names are alphabetically.

Reply:
I renamed it to keep that order. It makes more sense to keep it at this position.